app/scripts/lib/assertion.js

```diff
@@ -5,11 +5,12 @@
 define(function (require, exports, module) {
   'use strict';
 
+  var Duration = require('durationJs');
   var jwcrypto = require('vendor/jwcrypto');
   var P = require('lib/promise');
 
-  var CERT_DURATION_MS = 1000 * 60 * 60 * 6; // 6hrs
-  var ASSERTION_DURATION_MS = 1000 * 3600 * 24 * 365 * 25; // 25 years
+  var CERT_DURATION_MS =  new Duration('6h').milliseconds();
+  var ASSERTION_DURATION_MS = new Duration('52w').milliseconds() * 25; //25 years
```

Comment:
Doesn't 
```
if (assertionComponents.payload.exp < (Date.now() + LONG_LIVED_ASSERTION_DURATION - 5000)) {
      throw new Error('assertion should be long lived');
}
```

cover it? I mean, LONG_LIVED_ASSERTION_DURATION is set to the same amount as ASSERTION_DURATION_MS